src/core-itn/env/dev/terraform.tfvars

```diff
@@ -37,9 +37,10 @@ cidr_storage_italy    = ["10.3.5.0/24"]
 cird_redis_italy      = ["10.3.6.0/24"]
 cird_postgresql_italy = ["10.3.7.0/24"]
 
-cidr_cosmosdb_wallet_italy = ["10.3.8.0/24"]  #placeholder
-cird_redis_wallet_italy    = ["10.3.9.0/24"]  #placeholder
-cidr_storage_wallet_italy  = ["10.3.10.0/24"] #placeholder
+cidr_cosmosdb_wallet_italy         = ["10.3.8.0/24"]  #placeholder
```

Comment:
@diegolagospagopa @mamari90  could be useful do a az cli script that return the first available subnet ?
perhaps specifying the desired CIDR.
what do you think?
Otherwise today, the check is done by ✋ on az portal

Reply:
Right now i prefer to do it by hand, only to have a control. We can think in the future an automatic way

Reply:
@mamari90 Yes I can use `/27`